ggml/src/ggml-sycl/ggml-sycl.cpp

```diff
@@ -420,14 +421,13 @@ ggml_backend_sycl_buffer_cpy_tensor(ggml_backend_buffer_t buffer,
         return true;
     }
     return false;
-}
-catch (sycl::exception const &exc) {
-  std::cerr << exc.what() << "Exception caught at file:" << __FILE__
-            << ", line:" << __LINE__ << std::endl;
-  std::exit(1);
+    // TODO: Buffer is unused
```

Comment:
```suggestion
```
Looking at the other backends they also don't seem to use this `buffer` variable. I don't think the TODO is needed.